README.md

```diff
@@ -1 +1,79 @@
-# bounty
\ No newline at end of file
+# Bounty program for ooc bug fixing
+
+## Short explanation
+This is a bounty program which inspires interested programmers to solve bugs in the ooc language in exchange for fame, glory, and some money.
+
+## The history of ooc and the magic fork
+The ooc language was first released in 2009 and is a general-purpose high-level language which compiles to C code. Its compiler is called `rock`.
+
+In mid-2014 a fork was made of both the SDK, which is now greatly rewritten and extended, and the compiler. The latter has undergone some changes. Most notably, this fork does not use any garbage collection and introduces the `virtual` and `override` keywords.
+
+## Rules and practices
+All the code submitted through this process will be made available under the MIT license just as the rock compiler is.
+
+A solution is accepted if:
+1) It solves all the problems presented in its description
+2) It accurately compiles and runs the accompanied test files without crashes or memory leaks
+3) An inspection by us do not find any bugs
+4) The changes can be merged into the rock repo without any conflicts or introducing new problems
+
```

Comment:
I also think a solution should have a brief writeup about what they have done, how they did it and how it works. Maybe also some notes about how they found the solution. To make it easier for us (and ooc-lang people) to review it and also to log knowledge about rock.